### PR TITLE
use C-x r to get to the repl directly

### PR DESCRIPTION
**Pull request by noisesmith**
not totally sure on the mechanics, but it works

### CODE REVIEW

File: config/init.fnl [hunk 1]
```diff
@@ -1,4 +1,12 @@
 (local editor (require :polywell))
+(local state (require :polywell.state))
+
+(fn editor.cmd.goto-buffer [target]
+  "Create a command to go directly to a buffer"
+  (let [last-buffer (or (editor.last-buffer) target)]
+    (when (lume.find (editor.buffer-names) target)
+      (set state.last-buffer last-buffer)
```
Comment (technomancy) [inline config/init.fnl]: Dang; this actually shows a problem in polywell; it says that whoever calls `change-buffer` is responsible for setting `last-buffer`, but you can't do that without loading `polywell.state` which is intended to be a private module. That's my bad! Let's move the bit where last-buffer gets set into the change-buffer function so it's not the responsibility of the caller.

Comment (noisesmith) [reply config/init.fnl]: cool, sounds like the right thing

**Comment by noisesmith** [comment]
I updated change_buffer in the editor code to set the last buffer, and this means it can be used directly via partial in the key binding

I left the explicit setting of last-buffer in the polywell commands code because it sets a default (repl) if no last buffer was defined

**Review by technomancy — APPROVED** [review]
Perfect; thanks!

**Pull request merged**